workloads/router-perf-v2/env.sh

```diff
@@ -17,6 +17,9 @@ KUBE_BURNER_IMAGE=quay.io/cloud-bulldozer/kube-burner:latest
 TERMINATIONS=${TERMINATIONS:-"http edge passthrough reencrypt mix"}
 INFRA_TEMPLATE=http-perf.yml.tmpl
```

Comment:
It's my understand we have to change this ENV variables `INFRA_TEMPLATE`, but it always gets overridden right here no? Should you have:
`export INFRA_TEMPLATE=${INFRA_TEMPLATE:-http-perf.yml.tmpl}`

Same may go with `INFRA_CONFIG` if you think so.

Reply:
We set the environment variables in `env.sh` (this file), which is sourced on every execution of `ingress-performance.sh`.
`INFRA_CONFIG` never really needs to change, that's just the output filename after the vars are parsed.

Reply:
Got it. I was pointing out that it doesn't follow the pattern of most every other variable that it uses, e.g. `${VARIABLE:-"default"}`. I don't modify env.sh with my test runs, I use a .envrc source file per test configuration, which all other variables allow me to override.

I'm guessing this was previously hardcoded because it was never intended to change, but now, we have a reason to change `INFRA_TEMPLATE`. Is there a reason you don't want `INFRA_TEMPLATE` to follow this pattern? Keep it like this just makes it less configurable.

Reply:
You're right, there's no downside to changing it to match. I'm considering removing the `INFRA_CONFIG` var as well, as it is entirely irrelevant.